cardano-node/app/cardano-cli.hs

```diff
@@ -372,6 +393,15 @@ parseAddress opt desc =
         <> help desc
       )
 
+parseTxIn :: Parser TxIn
+parseTxIn = option (uncurry TxInUtxo
```

Comment:
This could be reformatted to:
```
parseTxIn :: Parser TxIn
parseTxIn =
  option
    ( uncurry TxInUtxo
        . Control.Arrow.first cliParseTxId
        <$> auto
      )
    ( long "txin"
        <> metavar "(TXID,INDEX)"
        <> help "Transaction input is a pair of an UTxO TxId and a zero-based output index."
      )
```

Reply:
It could be, but it seems fairly arbitrary.

I mean, how does one arrive at such a change, independently?

Reply:
I'm also not a big fan of the formatting here. I think the usage of horizontal space in this context is what bothers me. In fact, I think I would have even been okay with a minor variant of @deepfire's formatting:
```
parseTxIn :: Parser TxIn
parseTxIn = 
  option 
    (uncurry TxInUtxo 
      . Control.Arrow.first cliParseTxId
      <$> auto
    )
    ( long       "txin"
      <> metavar "(TXID,INDEX)"
      <> help    "Transaction input is a pair of an UTxO TxId and a zero-based output index."
    )
```

Reply:
@intricate, would you find it acceptable to have a minor variation on your option:
```
parseTxIn :: Parser TxIn
parseTxIn = 
  option 
  ( uncurry TxInUtxo 
    . Control.Arrow.first cliParseTxId
    <$> auto
  )
  ( long       "txin"
    <> metavar "(TXID,INDEX)"
    <> help    "Transaction input is a pair of an UTxO TxId and a zero-based output index."
  )
```
The difference is that this is compatible with the default way of `haskell-mode` indentation in Emacs -- whereas the previous option isn't (it's unavailable in the sequence of indentation options it provides).

Reply:
Vertically aligning code leads to git diffs with semantically null changes when the length of an identifier changes and other things are brought into alignment with it.

I currently rely heavily on git history and therefore highly value clean minimal commits with good commit messages. Anything that degrades the quality of git commits should be highly discouraged. Whoever maintains this code in the future will also rely heavily on git history. We should try and keep that history as clean and as useful as possible. This matters because we hope this code and the system it runs will be around for 50 years or more.

Reply:
@erikd, for the record -- I completely support that noble objective.  It's just that the original comment was unclear about the goal we were aiming for.